src/hooks/apps.ts

```diff
@@ -16,5 +18,11 @@ export default (queryConfig: QueryClientConfig) => {
         ...defaultQueryOptions,
         staleTime: CONSTANT_KEY_STALE_TIME_MILLISECONDS,
       }),
+    useMostUsedApps: (memberId?: UUID) =>
```

Comment:
You should not need to pass the memberId to the hook as this should be for the current member. See my other comment about the key.